package.json

```diff
@@ -78,13 +78,14 @@
     "copy-dir": "0.3.0",
     "coveralls": "2.11.12",
     "deepcopy": "0.6.3",
+    "flow-bin": "0.30.0",
```

Comment:
even though `grunt-flowbin` pulls this in for us, I'd like to keep it web-ext's `package.json` so that greenkeeper will auto-update our repo when new versions are released. It turns out web-ext was a bit behind.